src/ViewItem.js

```diff
@@ -74,6 +74,7 @@ class ViewItem extends React.Component {
       type: 'okapi',
       path: 'inventory/items/:{itemid}',
       POST: { path: 'inventory/items' },
+      resourceShouldRefresh: true,
```

Comment:
Why do you need this here? Our experience so far is that `resourceShouldRefresh` is generally a bit of a hack or a workaround for some other behaviour. As a result, it's always preferable to not rely on it. Barring that, I've found it immensely helpful when people document via a comment _why_ they're setting this to true. It tends to make future changes much simpler and accidental bugs much less likely.

Same comment about Line 107.

Reply:
In my experience, `resourceShouldRefresh` gets inserted whenever a PO's test pattern is "create a new entry for lookup-item-X in foo-settings, go to foo, edit a Foo, add comment to Jira that 'lookup-item-X was not present on Foo-edit page until I refreshed the browser'".

Reply:
Yes, I agree with Zak. Without applying the resourceShouldRefresh field, resource properties are not updated. Thus, when moving to the next item without reloading the page, the form displays the data of the previous item.